addons/core/translations/resources/en-us.yaml

```diff
@@ -362,3 +362,6 @@ managed-group:
     welcome:
       title: Welcome to Managed Groups
       description: Create and manage managed groups for an auth method.
+    accounts:
+      title: Welcome to Accounts
+      description: No accounts in this managed group.
```

Comment:
This shouldn't be part of empty state?

Reply:
This is used in a new `members` tab, thus isn't an empty state for managed group.